src/components/Pagination/Pagination.tsx

```diff
@@ -92,6 +90,16 @@ export class Pagination extends TsxComponent<Props> {
     this.$emit('page-change', this.selectedPage);
   }
 
+  private navigateToFirst() {
+    this.selectedPage = 1;
+    this.$emit('page-change', this.selectedPage);
+  }
+
+  private navigateToLast() {
+    this.selectedPage = this.numberOfPages;
+    this.$emit('page-change', this.selectedPage);
```

Comment:
Have you considered calling the event 'update:selectedPage' in order to support the .sync-modifier?

https://vuejs.org/v2/guide/components-custom-events.html#sync-Modifier

Reply:
Done